backend/database-collection-schemas/createAnnouncementCollection.py

```diff
@@ -26,44 +60,49 @@
     .set_key(APPWRITE_API_KEY)
 )
 
-
-createCollectionResult = database.create_collection(
-    "Announcements",  # Collection Name
-    ["*"],  # Read permissions
-    ["team:Admins"],  # Write permissions
-    [
-        {
-            "label": "created_at",
-            "key": "created_at",
-            "type": "numeric",
-            "required": True,
-            "array": False,
-        },
-        {
-            "label": "updated_at",
-            "key": "updated_at",
-            "type": "numeric",
-            "required": False,
-            "array": False,
-        },
-        {
-            "label": "title",
-            "key": "title",
-            "type": "text",
-            "required": True,
-            "array": False,
-        },
-        {
-            "label": "content",
-            "key": "content",
-            "type": "text",
-            "required": True,
-            "array": False,
-        },
-    ],
-)
-print(createCollectionResult)
-
+try:
+    createCollectionResult = database.create_collection(
+        "Announcements",  # Collection Name
+        ["*"],  # Read permissions
+        ["team:Admins"],  # Write permissions
+        [
+            {
+                "label": "created_at",
+                "key": "created_at",
+                "type": "numeric",
+                "required": True,
+                "array": False,
+            },
+            {
+                "label": "updated_at",
+                "key": "updated_at",
+                "type": "numeric",
+                "required": False,
+                "array": False,
+            },
+            {
+                "label": "title",
+                "key": "title",
+                "type": "text",
+                "required": True,
+                "array": False,
+            },
+            {
+                "label": "content",
+                "key": "content",
+                "type": "text",
+                "required": True,
+                "array": False,
+            },
+        ],
+    )
+    print(createCollectionResult)
+    print(createCollectionResult["$id"])
+    with open("announcementCollectionID.txt", "w") as f:
+        print(createCollectionResult["$id"], file=f)
+except Exception as e:
+    print(e)
+    sys.exit(1)
 # Create some fake data
 data = [
```

Comment:
better would have been
"announcementEntries"
or
"announcementTuples"

Reply:
There could be failure message returned. In success case, this will return:
```json
{
    "$id": "5e5ea5c16897e",
    "$collection": "5e5ea5c15117e",
    "$read": [
        "role:all"
    ],
    "$write": [
        "user:608f9da25e7e1"
    ]
}
```
So I don't think there should be need for changing the var name here.